docs/source/ops/index.rst

```diff
@@ -30,6 +30,9 @@ Still, Daml applications may be affected in the following ways:
 - Command deduplication and command tracker retention should always configured in such a way, that the associated windows don't overlap with the pruning window, so that their operation is unaffected by pruning.
 - Pruning may affect the behavior of Ledger API calls that allow to read data from the ledger: see the next sub-section for more information about API impacts.
 
+.. warning::
+   Participant pruning and the use of *divulged contracts* during interpretation are in conflict with each other. A participant may never learn about the archival of a divulged contract and has no reliable way to detect when a divulged contract can safely be removed from the participant node, resulting in an ever growing participant node database and failed command submissions due to a potentially outdated view of the contract on the participant. Users are advised to not use divulged contracts in their Daml workflows.
```

Comment:
```suggestion
  Participants may know of contracts for which they don't know the current activeness status. This happens through :ref:`divulgence <da-model-divulgence>` where a party learns of the existence of a contract without being guaranteed to ever see its archival. Such contracts are pruned by the feature described on this page as not doing so could easily lead to an ever growing participant state.
  
During command submission, parties can currently resolve the contract ids of divulged contracts. This is mutually incompatible with the pruning behaviour described above. Daml code that may end up running on pruned participants should therefore never rely on that behaviour.```

Reply:
I don't know whether this is truly an improvement, but I found it a bit hard to read.

Reply:
Thank you!